log_bucket/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "cdn_logs_path" {
```

Comment:
for these can you a `type = string` that helps with Terraform 0.12+ to allow type checking of input variables. You get better error messages